src/pubtools/_pulp/tasks/copy_repo.py

```diff
@@ -102,6 +102,14 @@ class RepoCopy(object):
     """The repo to which content was copied."""
 
 
+class partpartial(partial):
```

Comment:
Minor: I am not really fond of the name, because to me it implies a *weaker* kind of partial ("part partial"), while this is actually *stronger* than a usual partial object. It freezes *all* the arguments, while I think partial's name comes from the fact that it's typically used to *partially* freeze/bind a function's arguments.

Not a big deal obviously, given it's not public API.

Some other ideas for names: `freeze_arguments`, `bind_arguments`, `full_partial`.

Reply:
Thanks for the insight. 
"Part partial" came from the fact that it's not providing all the features of a "partial", hence the name. However, suggested names sounds more meaningful.